engine/apps/public_api/urls.py

```diff
@@ -34,4 +34,5 @@
     optional_slash_path("info", views.InfoView.as_view(), name="info"),
     optional_slash_path("make_call", views.MakeCallView.as_view(), name="make_call"),
     optional_slash_path("send_sms", views.SendSMSView.as_view(), name="send_sms"),
+    optional_slash_path("direct_paging", views.DirectPagingView.as_view(), name="direct_paging"),
```

Comment:
Since we call it "Escalation" in the UI, and the slack command is "escalate", wondering if the endpoint should follow that line? (`/escalation`?)

Reply:
good point 👍 addressed in https://github.com/grafana/oncall/pull/4815/commits/b13791e2925965a0e8b079fb95172a844fdf1dbc